runtime.md

```diff
@@ -99,7 +97,7 @@ This operation MUST return the state of a container as specified in the [State](
 `create <container-id> <path-to-bundle>`
 
 This operation MUST [generate an error](#errors) if it is not provided a path to the bundle and the container ID to associate with the container.
-If the ID provided is not unique across all containers within the scope of the runtime, or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
+If the ID provided is not unique across all containers within the [scope of the container](#scope-of-a-container), or is not valid in any other way, the implementation MUST [generate an error](#errors) and a new container MUST NOT be created.
```

Comment:
> ... if an ID is unique or not, would be compared with other containers', which is out of the scope of A container.

I expect

    $ funC create $ID ...

to be checking for an exiting $ID container using the same funC-specific state-sharing mechanism that

   $ fun state $ID

and other operations on that container are using.  I think the "Scope of a Container" section is really about the scope of this state-sharing (and I'm happy to rename that section if it helps clarify this).

Do you see a point in distinguishing between "state-sharing for ID deduping" and "state-sharing for everything else"?